simphony_metadata.yml

```diff
@@ -419,3 +419,28 @@ CUDS_KEYS:
   ELECTROSTATIC_MODEL:
     parent: CUBA.PHYSICS_EQUATION
     definition: Electrostatic model
+    physics_equation: [MOLECULAR_DYNAMICS]
+
+  BASIS:
```

Comment:
A LATTICE has an attribute VECTORS, propose to remove lattice_vectors to follow better the EAV model!